dev/org.eclipse.codewind.ui/src/org/eclipse/codewind/ui/internal/actions/CodewindInstall.java

```diff
@@ -0,0 +1,213 @@
+/*******************************************************************************
+e * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ *	 IBM Corporation - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.codewind.ui.internal.actions;
+
+import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.util.concurrent.TimeoutException;
+
+import org.eclipse.codewind.core.internal.InstallUtil;
+import org.eclipse.codewind.core.internal.Logger;
+import org.eclipse.codewind.core.internal.ProcessHelper.ProcessResult;
+import org.eclipse.codewind.ui.CodewindUIPlugin;
+import org.eclipse.codewind.ui.internal.messages.Messages;
+import org.eclipse.codewind.ui.internal.wizards.NewCodewindProjectWizard;
+import org.eclipse.codewind.ui.internal.wizards.WizardLauncher;
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.core.runtime.jobs.Job;
+import org.eclipse.jface.wizard.Wizard;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.MessageBox;
+import org.eclipse.swt.widgets.Shell;
+
+
+public class CodewindInstall {
+	
+	private static final String CODEWIND_NOT_INSTALLED = "Codewind is not installed";
+	
+	public static boolean isCodewindInstalled() throws InvocationTargetException {
```

Comment:
This method is a bit confusing because it is called "isCodewindInstalled" but it returns true if it is not installed and false if it is installed:
if (isCodewindInstalled.equals(CODEWIND_NOT_INSTALLED)) {
         return true;
} else {
	return false;
}
Can it be changed to return true if Codewind is installed?